access/discord/Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION=14.3.3
+VERSION?=14.3.3
```

Comment:
> This is only needed if you want environment variables to override make variables

This is why I changed the line. If I remember correctly, during the build process we're invoking `make` in such a way that args don't get passed down properly. Rather than try and chase down everywhere this is occurring and fix it I decided that this relatively minimal change was the best approach for this work.